icechunk/src/config.rs

```diff
@@ -20,9 +20,7 @@ pub enum ObjectStoreConfig {
     LocalFileSystem(PathBuf),
     S3Compatible(S3CompatibleOptions),
     S3(S3CompatibleOptions),
-    Gcs {
-        // TODO:
-    },
+    Gcs(Option<Vec<(String, String)>>),
```

Comment:
Can we give this more structure? Are there mandatory fields? Why `Option` and not empty `Vec`?